### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by tamlyn** [DevExpress/testcafe-live#9]
When I run TestCafe Live, the test runs fine but then as soon as I make a change in a file it crashes:

```
Make changes in the source files or press ctrl+r to restart test run.

Stopping TestCafe Live...
.../node_modules/lodash/lodash.js:12148
    function isSymbol(value) {
                     ^

RangeError: Maximum call stack size exceeded
    at isSymbol (.../node_modules/lodash/lodash.js:12148:22)
    at isKey (.../node_modules/lodash/lodash.js:6320:28)
    at castPath (.../node_modules/lodash/lodash.js:4471:14)
    at hasPath (.../node_modules/lodash/lodash.js:6132:14)
    at Function.has (.../node_modules/lodash/lodash.js:13150:32)
    at Graph.hasNode (.../node_modules/graphlib/lib/graph.js:169:12)
    at ModulesGraph.addNode (.../node_modules/testcafe-live/lib/file-watcher/modules-graph.js:32:24)
    at children.forEach.child (.../node_modules/testcafe-live/lib/file-watcher/modules-graph.js:25:22)
    at Array.forEach (<anonymous>)
    at ModulesGraph._updateChildren (.../node_modules/testcafe-live/lib/file-watcher/modules-graph.js:21:22)
```

I'm using Node 8.9 and TestCafe 0.18.6 on OSX 10.11.

**Comment by AlexanderMoskovkin** [comment]
Hi @tamlyn,

Thanks for reporting. I'll investigate it. You can try the `0.1.1` version for now. It shouldn't have this issue.
If it's appropriate for you could you please send me your test files that you try to run? You can do it privately to the email from my GitHub profile. I don't need the whole code but you can send only files structure and `import`  directives from all files. It can help me a lot to identify the reason of the issue and I will try to fix it asap.

**Comment by tamlyn** [comment]
Actually it's open source so you can clone it yourself: https://gitlab.coko.foundation/pubsweet/pubsweet-starter

Instructions assume you have the `pubsweet` CLI tool installed globally but you can use the local one instead via `npx`.

**Comment by AlexanderMoskovkin** [comment]
Thanks. I'll check it

**Comment by AlexanderMoskovkin** [comment]
Fixed in `0.1.3`
